teuthology/task/ssh_keys.py

```diff
@@ -15,6 +16,31 @@
 log = logging.getLogger(__name__)
 ssh_keys_user = 'ssh-keys-user'
 
+
+def timestamp(format_='%Y-%m-%d_%H:%M:%S:%f'):
+    """
+    Return a UTC timestamp suitable for use in filenames
+    """
+    return datetime.utcnow().strftime(format_)
+
+
+def backup_file(remote, path, sudo=False):
```

Comment:
I'm not sure how this should be tested and, since it's likely a temporary addition, I don't care too much about making it the first task with unit tests.